pkg/deployments/tasks/20220318-batch-relayer-v2/input.ts

```diff
@@ -0,0 +1,28 @@
+import Task from '../../src/task';
+
+export type BatchRelayerDeployment = {
+  Vault: string;
+  wstETH: string;
+};
+
+const Vault = new Task('20210418-vault');
+
+export default {
+  Vault,
+  // wstETH is only deployed on mainnet and kovan
+  mainnet: {
+    wstETH: '0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0',
+  },
+  kovan: {
+    wstETH: '0xa387b91e393cfb9356a460370842bc8dbb2f29af',
+  },
+  rinkeby: {
+    wstETH: '0x0000000000000000000000000000000000000000',
+  },
+  polygon: {
+    wstETH: '0x0000000000000000000000000000000000000000',
+  },
+  arbitrum: {
+    wstETH: '0x0000000000000000000000000000000000000000',
+  },
```

Comment:
My intention was to leave them because they are fully specified for other networks and there is nothing stopping us from later deploying to other networks. Do you really think this matters?

Reply:
We should at least mention the fact that we only plan to do it (or have done it) on Polygon _somewhere_ - it otherwise looks like the deployments for the other networks are missing.